mysql-test/suite/multi_source/gtid_slave_pos.test

```diff
@@ -118,8 +118,11 @@ set default_master_connection = 'slave1';
 STOP SLAVE;
 --source include/wait_for_slave_to_stop.inc
 set default_master_connection = 'slave2';
+--let $slave_sql_errno= 1942
+--source include/wait_for_slave_sql_error.inc
 STOP SLAVE;
---source include/wait_for_slave_to_stop.inc
+--let $rpl_only_running_threads= 1
```

Comment:
You should be able to remove the above `STOP SLAVE` as it is now redundant with the addition of `source include/stop_slave.inc`